pkgs/os-specific/linux/pentablet-drivers/xp-pen-deco-01-v2-driver/default.nix

```
@@ -0,0 +1,64 @@
+{ lib, stdenv, fetchzip, libusb, glibc, libGL, xorg, qtx11extras, wrapQtAppsHook, autoPatchelfHook, 
+libX11, libXtst, libXi, libXrandr, libXinerama, ... }:
+
+let
+  dataDir = "var/lib/xppend1v2";
+in
+stdenv.mkDerivation {
+  pname = "xp-pen-deco-01-v2-driver";
+  version = "3.2.3.220323-1";
+  src = fetchzip {
+    url = "https://www.xp-pen.com/download/file/id/1936/pid/440/ext/gz.html#.tar.gz";
```

You could add `name` here as well, so that logs show you're downloading that instead of `gz.html#.tar.gz`. 
```suggestion
    name = "xp-pen-deco-01-v2-driver-${version}.tar.gz";
    url = "https://www.xp-pen.com/download/file/id/1936/pid/440/ext/gz.html#.tar.gz":
```